tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario: group member lists the file shared with them when auto-sync is disabled
```

Comment:
This scenario is for project space so would be better to indicate that in scenario title
```suggestion
  Scenario: group member lists the file shared with them when auto-sync is disabled
```